server/tests/test_parameterval_views.py

```diff
@@ -37,8 +37,17 @@ def test_parameterval_detail_get(self):
         self.assertEqual(str_val['parameter_spec']['name'], 'StringParam')
         self.assertEqual(str_val['parameter_spec']['id_name'], 'stringparam')
         self.assertEqual(str_val['parameter_spec']['type'], ParameterSpec.STRING)
+        self.assertEqual(str_val['parameter_spec']['placeholder'], 'placeholder')
```

Comment:
I generally try to avoid using test text that is the same as other text. E.g. use something other than 'placeholder' for the placeholder text. This avoids both human confusion and certain potential bugs.